aurora/bag_transfer/test/test_users.py

```diff
@@ -0,0 +1,117 @@
+import random
+from unittest.mock import patch
+
+from bag_transfer.models import Organization, User
+from django.test import TestCase
+from django.urls import reverse
+
+
+class UserTestCase(TestCase):
+    fixtures = ["complete.json"]
+
+    def setUp(self):
+        self.client.force_login(User.objects.get(username="admin"))
+
+    def assert_status_code(self, method, url, data, status_code):
```

Comment:
This is more of a maybe-add-this-as-an-issue thing, but could this be added to `helpers.py` and incorporated into the other tests?

Reply:
@bonniegee yes absolutely - I have a not-yet-filed issue about that. I was waiting to see if other things surfaced while re-writing tests.